src/partials/Program.HelpText.cs

```diff
@@ -53,7 +53,7 @@ internal partial class Program
     -p, --path                Absolute path to install location
     -n, --name                The name of the extra to install. Required
                                  Values: DonkeyKong, RadarScope, jtbubl, jtcps1, jtcps15,
-                                         jtcps2, jtpang, toaplan2_c, jtc16_c
+                                         jtcps2, jtpang, toaplan2_c, jtc16_c, sgb2, sgbvw

```

Comment:
i think a good future improvement here would be we can add a --list and --info option. so we don't need to hardcode the list here, but let the user display it dynamically, and then they can also get the description via --info

Reply:
So --list would list out the values and maybe the descriptions?
And --info would do what?

Reply:
yeah didn't think about the required name. don't have to worry about it right now

Reply:
Turns out the required attribute doesn't display an error message when it's missing, just terminates. So not using it is actually better. Sadly.

Reply:
Got both list and info added as well as a fix for the newer cli commands erroring out when a path isn't specified.